README.md

```diff
@@ -107,6 +107,18 @@ implementation("com.squareup.okhttp3:okhttp:4.9.0")
 
 Snapshot builds are [available][snap]. [R8 and ProGuard][r8_proguard] rules are available.
 
+Also, we have a [bill of materials (BOM)][bom] available to help you keep OkHttp artifacts up to date and be sure about versions compatibility.
+
+```kotlin
+    dependencies {
+       // define a BOM and its version
+       implementation(platform("com.squareup.okhttp3:okhttp-bom:4.9.0"))
+       
+       // define any required OkHttp artifacts without verion
```

Comment:
typo: version

Reply:
Good catch. I will fix it. Wanted to add the same snippet for snapshot verisons

Reply:
Already fixed it. Not sure why I mentioned it to be honest. Thanks for adding this.